frontend/src/components/transactions/Transactions.tsx

```diff
@@ -45,6 +45,7 @@ const TransactionsWrapper: FC<OuterProps> = ({
   );
   return (
     <TransactionsList
+      key={accountId || blockHash}
```

Comment:
I am a bit surprised to see `key` used in this context. Are you sure it is needed here?

Reply:
I just want to add that there are three transactions lists:

1. Transactions on 'Transactions' page (which is independent from any props except of `paginationIndexer` at least);
2. Transactions on Blocks page (which depends on `blockHash`);
3. Transactions on Accounts page (which depends on `accountId`);

So from my point of view even if you want to add `key` than `accountId` and `blockHash` is not enough. Why do we need it here?

Reply:
> Transactions on 'Transactions' page (which is independent from any props except of paginationIndexer at least);

It never changes its source (`fetch` function, technically speaking), so it's not the case.

> Transactions on Blocks page (which depends on blockHash)
Transactions on Accounts page (which depends on accountId)

Both 'Blocks' and 'Accounts' page transaction lists use this exact component.

> I am a bit surprised to see key used in this context. Are you sure it is needed here?

Sure, we basically remount the whole component if we need to change transaction list source.
https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html#recommendation-fully-uncontrolled-component-with-a-key